src/Page-Components/Article.jsx

```diff
@@ -0,0 +1,51 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import "../css/Style.css";
+import "../css/Article.css";
+import Loading from "./Loading";
+import fetchApiData from "../api";
+
+const Article = () => {
+  const [article, setArticle] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    fetchApiData.get(`/articles/${article_id}`).then(({data}) => {
+      setArticle(data.article);
+    });
+    setIsLoading(false);
+  }, []);
+
+  if (isLoading) {
+    return <Loading />;
+  } else {
+    return (
+    <>
+      <main className="page-container">  
+        <article className="singleArticle">
+          <h2 className="text-black font-bold text-xl my-3">{article.title}</h2>
+          <img id='singleArticleImg' src={article.article_img_url} alt="" />
+          <section id="articleInfo">
+            <p className='text-black font-bold'>Topic: <span className='text-black font-medium'>{article.topic}</span></p>
+            <p className='text-black font-bold '>Author: <span className='text-black font-medium'>{article.author}</span></p>
+            <p className='text-black font-bold'>Posted: <span className='text-black font-medium'>{article.created_at}</span></p>
```

Comment:
You might want to consider using a library like [Day.js](https://day.js.org/) to make your timestamps a bit more readable 🔍